contracts/root/RootChain.sol

```diff
@@ -0,0 +1,106 @@
+pragma solidity ^0.5.5;
+
+import { RLPReader } from "solidity-rlp/contracts/RLPReader.sol";
+import { Ownable } from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import { IRootChain } from './IRootChain.sol';
+import { RootChainHeader, RootChainStorage } from './RootChainStorage.sol';
+import { IStakeManager } from './stakeManager/IStakeManager.sol';
+import { Registry } from './Registry.sol';
+
+contract RootChain is RootChainStorage, IRootChain {
```

Comment:
I think here `RootChainStorage` will be connected to `RootChain` not proxy. Same with `WithdrawManager`: `WithdrawManagerStorage` is attached to both `WithdrawManager` and `WithdrawManagerProxy` but using `WithdrawManager`'s version of storage. Or I am missing something.

Reply:
As discussed on phone, need to be in both because of how delegate call works.